job-definition-executor/src/main/java/de/wacodis/coreengine/executor/process/wps/WPSProcess.java

```diff
@@ -138,127 +125,6 @@ public ProcessOutputDescription execute(ProcessContext context) throws Execution
         }
     }
 
-    private WPSProcessInput buildExecuteRequest(ProcessContext context) {
```

Comment:
@arnevogt I have extracted all these methods into their own class to make them better testable.